elpy.el

```diff
@@ -2050,17 +2063,27 @@ This requires Python 2.7 or later."
 (put 'elpy-test-discover-runner 'elpy-test-runner-p t)
 
 (defun elpy-test-django-runner (top file module test)
-  "Test the project using the Django discover runner.
+  "Test the project using the Django discover runner,
+   or with manage.py if elpy-test-django-with-manage is true.
 
 This requires Django 1.6 or the django-discover-runner package."
   (interactive (elpy-test-at-point))
   (if module
       (apply #'elpy-test-run
              top
-             (append elpy-test-django-runner-command
-                     (list (if test
-                               (format "%s.%s" module test)
-                             module))))
+             (append
+              ;; if we want to use manage.py, get the root directory where it is.
+              (if elpy-test-django-with-manage
+                         (append
+                          (list (concat
+                           (expand-file-name (locate-dominating-file (elpy-project-root) (car elpy-test-django-runner-manage-command)))
```

Comment:
On the other hand, this line could really be split up on multiple lines to make it clearer what is happening here. (General rule of thumb, a line break can be useful after a number of consecutive closing parens).